nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/main.nf

```diff
@@ -159,6 +159,26 @@ if(params.email) summary['E-mail Address'] = params.email
 log.info summary.collect { k,v -> "${k.padRight(15)}: $v" }.join("\n")
 log.info "========================================="
 
+
+def create_workflow_summary(summary) {
+
+    def yaml_file = workDir.resolve('workflow_summary_mqc.yaml')
+    yaml_file.text  = """
+    id: '{{ cookiecutter.pipeline_name }}-summary'
+    description: " - this information is collected when the pipeline is started."
+    section_name: 'nfcore/{{ cookiecutter.pipeline_slug }} Workflow Summary'
```

Comment:
Ok, and this one should be `pipeline_name ` 😆 because it's human readable

Also `nf-core`, not `nfcore` here.